jetty-server/src/main/config/modules/requestlog.mod

```diff
@@ -16,6 +16,9 @@ etc/jetty-requestlog.xml
 [files]
 logs/
 
+[ini]
+jetty.requestlog.dir?=logs
```

Comment:
I don't think this change is necessary.
There's already a `default="logs"` in the xml.

Also, don't define (in XML files) a shorthand reference like this, it would be best to include the `${jetty.base}` to have

```
[ini]
jetty.requestlog.dir?=${jetty.base}/logs
```

Reply:
Its not a necessary change but it is helpful.

Say you uncommented the `jetty.requestlog.filePath` line, then it doesn't work unless you also uncomment the  `jetty.requestlog.dir` line.

With this default value, if you want to make the change:
```
## File path
jetty.requestlog.filePath=/tmp/${jetty.requestlog.dir}/yyyy_mm_dd.request.log
```

It just works without uncommenting the additional line.

Reply:
I think it may be better to just change the comment to
```
 ## File path 
# jetty.requestlog.filePath=/tmp/logs/yyyy_mm_dd.request.log
```

and thus avoid the need for the `?=` in ini

Reply:
If we do that there will be no relation between the two properties, if you uncomment this line it overrides the other property and makes it redundant.

I can just remove the `?=` and it will work the same as it did before.